tower-balance/Cargo.toml

```diff
@@ -48,7 +48,7 @@ quickcheck = { version = "0.6", default-features = false }
 tokio = "=0.2.0-alpha.4"
 tokio-executor = "=0.2.0-alpha.4"
 tokio-test = "=0.2.0-alpha.4"
-tower-buffer = { version = "=0.3.0-alpha.1", path = "../tower-buffer" }
+tower-buffer = { version = "=0.3.0-alpha.1a", path = "../tower-buffer" }
```

Comment:
My vote is to just call this `-alpha.2`.

Reply:
Only reason I want to do `1a` is because I want people to know this will work with all `alpha.1` if we go to `alpha.2` then I need to release all the crates again or we will have a mix of 2 and 1.

Reply:
This isn't a breaking change either it is just a patch :shrug: